src/main/java/com/ardaslegends/service/war/WarService.java

```diff
@@ -95,43 +99,94 @@ public War createWar(CreateWarDto createWarDto) {
             throw WarServiceException.alreadyAtWar(attackingFaction.getName(), defendingFaction.getName());
         }
 
-        // Get Roles
-        if(attackingFaction.getFactionRole() == null) {
-            attackingFaction.setFactionRole(fetchFactionRole(attackingFaction));
-        }
-        if(defendingFaction.getFactionRole() == null) {
-            defendingFaction.setFactionRole(fetchFactionRole(defendingFaction));
-        }
-
```

Comment:
this indeed was useless 🤣
thanks for removing it

Reply:
Np cheffe